srcpkgs/pcc/template

```diff
@@ -1,24 +1,31 @@
 # Template file for 'pcc'
 pkgname=pcc
-version=1.1.0
-revision=2
+_snapshot=20190420
+_distver=1.1.0
+version=${_distver}.${_snapshot}
+revision=1
+wrksrc=${pkgname}-${_distver}.BETA
 build_style=gnu-configure
-configure_args="--enable-tls"
+configure_args="--enable-tls --enable-native"
 hostmakedepends="flex"
 makedepends="pcc-libs"
-depends="pcc-libs>=${version}"
+depends="binutils pcc-libs>=${version}"
 short_desc="Portable C Compiler"
-homepage="http://pcc.ludd.ltu.se/"
-license="BSD"
 maintainer="Juan RP <xtraeme@voidlinux.org>"
-distfiles="http://pcc.ludd.ltu.se/ftp/pub/pcc-releases/pcc-$version.tgz"
-checksum=d9e3fed84c767dcddefb6b1f854d5da8921f3f65d8bd02997743412f41474d64
-archs="i686 x86_64"
+license="2-clause-BSD"
+homepage="http://pcc.ludd.ltu.se/"
+distfiles="http://pcc.ludd.ltu.se/ftp/pub/pcc-beta/pcc-${_distver}.BETA-${_snapshot}.tgz"
+checksum=a114dfca04b8b0559ea5d67b87d2d0fcb2bae2b4d9b9ba16ebc27960b921c812
+nocross=1
 
 pre_configure() {
 	sed -i '/GNUC_MINOR/s,3,1,g' cc/cc/cc.c cc/driver/platform.c
 }
+pre_build() {
+	cp common/*.[ch] mip
+}
 post_install() {
+	vlicense ${XBPS_DISTDIR}/srcpkgs/pcc-libs/files/LICENSE
```

Comment:
XBPS_SRCPKGDIR can be used to cut the `srcpkgs/` part but it is preferable that the license is fetched on distfiles if they are not present in the tarball